src/utils/query.js

```diff
@@ -2,6 +2,14 @@ import { actionTypes } from '../constants'
 
 const { UNSET_LISTENER } = actionTypes
 
+const tryParseToNumber = (value) => {
+  const result = Number(value)
```

Comment:
According to [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number):

> If the argument cannot be converted into a number, it returns NaN.

And effectively, i've just tested in Chrome, Firefox and Node and `Number(undefined)` returns `NaN`. So, `tryParseToNumber` would return the undefined itself.